### ISSUE TITLE
source tarball available on PyPI is missing zran.h?

### ISSUE THREAD

**Issue by boegel** [pauldmccarthy/indexed_gzip#4]
I was trying to build indexed_gzip from source, but I ran into the following error message:

```
   indexed_gzip/indexed_gzip.c(457): catastrophic error: cannot open source file "zran.h"
      #include "zran.h"
```

It turns out that `zran.h` isn't included in the source tarball available through PyPI, but it is available in the repository, and hence also in the tarball provided by GitHub for the tagged release.

I presume that's not intentional?

**Comment by pauldmccarthy** [comment]
Aah - no that is not intentional, thanks for pointing it out. I am planning to do a new release in the next day or so, so will make sure this is fixed!

**Comment by pauldmccarthy** [comment]
This should be resolved as of the latest version, 0.6.1 - thanks!
